app/manifest.json

```diff
@@ -1,36 +1,37 @@
 {
     "name": "__MSG_appName__",
-    "version": "0.10.0",
+    "version": "1.0.2",
     "manifest_version": 2,
     "description": "__MSG_appDescription__",
     "icons": {
         "16": "images/icon-16.png",
         "128": "images/icon-128.png"
     },
+    "applications": {
+        "gecko": {
+            "id": "addon@fullyfeedly"
```

Comment:
Out of curiosity, is it required to package the application for Firefox?

Reply:
This is more necessary to debug the application in Firefox since the local storage API requires the application ID to function. It's also required for publishing the extension to addons.firefox.com.